python/graphstorm/gconstruct/utils.py

```diff
@@ -141,6 +144,14 @@ def _estimate_sizeof(data):
 
     return data_size
 
+def generate_hash():
+    """ Generate unique hashcode
+    """
+    random_uuid = uuid.uuid4()
```

Comment:
Can we use uuid directly?

Reply:
Let me check if we can directly use str(random_uuid), uuid here is an object, so we need to use the string conversion